examples/test-app/steps/Start/template.html

```diff
@@ -0,0 +1,24 @@
+{% extends "look-and-feel/layouts/start_page.html" %}
+
+{% set title='Apply for Divorce' %}
+{% set phase='ALPHA' %}
+{% set feedbackLink='https://github.com/hmcts/look-and-feel/issues/new' %}
+
+{% block head -%}
+  <link href="{{ asset_path }}main.css" media="screen" rel="stylesheet" />
```

Comment:
I realise that `{{ asset_path }}main.css` relates to `look-and-feel` and has nothing to do with this PR bit it would read far better if it had a forward slash like so:  `{{ asset_path }}/main.css`. Maybe there's a good reason, however, it looks odd.

Reply:
perhaps raise as an issue?

Reply:
I'll do that no problem.